static/src/js/components/Tour/SearchTour.scss

```diff
@@ -0,0 +1,157 @@
+:root {
+  --color-white: #ffffff;
+  --color-black: #000000;
+  --color-heading: #0063A6;
+  --color-gray: #9a989a;
+  --color-light-gray: #f5f5f5;
+  --font-mono: "DM Mono", SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
```

Comment:
We should have a font sass variable. If we dont, we should add one. We prefer sass variables over css variables for consistency.